Wikipedia/UI-V5/WMFArticleContainerViewController.m

```diff
@@ -279,6 +265,11 @@ - (void)setupToolbar {
 }
 
 - (void)updateToolbarItemsIfNeeded {
+    
+    if (!self.saveButtonController) {
```

Comment:
bikeshedding? I'm working on other tickets and pressing to get the alpha out, so if it actually doesn't have an impact…